src/taco-cli/cli/kit.ts

```diff
@@ -721,7 +721,7 @@ class Kit extends commands.TacoCommandBase {
         });
     }
 
-    private static select(commandData: commands.ICommandData): Q.Promise<tacoUtility.ICommandTelemetryProperties> {
```

Comment:
What's the reasoning behind making these instance methods rather than static methods?